demos/graph-classification/supervised-graph-classification.ipynb

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
<p>If you were so inclined, you could use python's native rounding-formatting:</p><p> </p><pre class="ql-syntax" spellcheck="false">print(
&nbsp; &nbsp; f"Accuracy over all folds mean: {np.mean(test_accs)*100:.1} and std: {np.std(test_accs)*100:.1}"
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>